src/docker/custom.rs

```diff
@@ -22,6 +23,40 @@ pub enum Dockerfile<'a> {
     },
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+pub enum PreBuild {
```

Comment:
Should we allow `pre-build = "command" to exist? Or should that only be reserved for scripts? Say someone has a `dpkg` in the Docker context: shouldn't we avoid having conflicts.

Reply:
I think it's a good idea just to avoid ambiguity. It will make serializing it a bit more difficult, however.